### ISSUE TITLE
Consider making the project a web app as well served using GitHub pages...

### ISSUE THREAD

**Issue by HazemSayad** [iconforest/icon_forest#5]
The project is nice, i love the idea but i sure won't download an app on my phone to browse icons for my projects. I think that this should be available over web as well. Devs can simply bookmark a link, and browse icons in the web browser on the same pc they are developing on. So consider making a web version maybe over GitHub pages...

**Comment by DavidOrakpo** [comment]
Exactly. I can't fathom why an app was used for this process instead of being on web

**Comment by sucanabo** [comment]
yah, I agree. A website or a vibrant document is essential
